posthog/hogql_queries/utils/query_date_range.py

```diff
@@ -250,22 +249,6 @@ def to_placeholders(self) -> Dict[str, ast.Expr]:
             else self.date_from_as_hogql(),
         }
 
-    def to_properties(self, field: Optional[List[str]] = None) -> List[ast.Expr]:
-        if not field:
-            field = ["timestamp"]
-        return [
-            ast.CompareOperation(
-                left=ast.Field(chain=field),
-                op=CompareOperationOp.LtEq,
-                right=self.date_to_as_hogql(),
```

Comment:
This should be `date_to` here as well, but the whole method is unused. Hence removed.